test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/CcmBridge.java

```diff
@@ -327,6 +327,10 @@ private static File createTempStore(String storePath) {
     } catch (IOException e) {
       logger.warn("Failure to write keystore, SSL-enabled servers may fail to start.", e);
     }
+    // ensure the temp store files are deleted when the JVM exits
+    if (f != null) {
```

Comment:
This could be in a finally block for the preceding try, although I don't expect runtime exceptions to come out of that block, so this is fine as is too.

Reply:
While at it, I suggest rewriting the whole method:

```
  private static File createTempStore(String storePath) {
    File f = null;
    try (OutputStream os = new FileOutputStream(f = File.createTempFile("server", ".store"))) {
      f.deleteOnExit();
      Resources.copy(CcmBridge.class.getResource(storePath), os);
    } catch (IOException e) {
      logger.warn("Failure to write keystore, SSL-enabled servers may fail to start.", e);
    }
    return f;
  }
```
`Resources` is actually `com.datastax.oss.driver.shaded.guava.common.io.Resources`.